tools/docker/nginx/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Build eda-frontend
+FROM docker.io/node:16-alpine AS ui-builder
+WORKDIR /app
+
+COPY . $WORKDIR
+
+RUN cd ui && npm install && npm run build
+
+FROM nginx
```

Comment:
Use the FQDN for the image.

Reply:
I recommend registry.access.redhat.com/ubi8/nginx-120 but it would require to change the other default paths.

Reply:
I'll look at that image as long as we can just over ride the default.conf we should be fine I think

Reply:
@ddonahue007 Is this changing going in?

Reply:
This will happen on another PR. We will want TLS features when we move to that we can move to the correct nginx image.